tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1063,7 +1063,7 @@ def crop_to_bounding_box(image, offset_height, offset_width, target_height,
     'image.resize_with_crop_or_pad',
     v1=['image.resize_with_crop_or_pad', 'image.resize_image_with_crop_or_pad'])
 @dispatch.add_dispatch_support
-def resize_image_with_crop_or_pad(image, target_height, target_width):
+def resize_image_with_crop_or_pad(image, target_height, target_width, mode="CONSTANT", name=None, constant_values=0):
```

Comment:
There are two places in the docstring that need to be updated. Where it says `"...padding it evenly with zeros"` and `"...this op centrally pads with 0 along..."`. Could you update the description please?

Reply:
Sorry, this has been resolved to properly reference 'constant_values' instead of 0